lib/builderator/tasks/packer.rb

```diff
@@ -24,13 +25,17 @@ def configure(profile = :default)
         Config.profile.use(profile)
 
         invoke Tasks::Version, :current, [], options
+        # TODO: There's a feature here where if you run configure from the
+        # command line and in debug mode, you will get a security group
+        # created but not destroyed.  This may or may not be expected or desired.
         puts Interface.packer.render if options['debug']
       end
 
       desc 'build [PROFILE=default *ARGS]', 'Run a build with the installed version of packer'
       def build(profile = :default, *args)
         invoke :configure, [profile], options
         run_with_input "#{Interface.packer.command} build - #{ args.join('') }", Interface.packer.render
+        Util.remove_security_group(Config.aws.region, Interface.packer.security_group_id)
```

Comment:
addressed in https://github.com/rapid7/builderator/pull/116/commits/4974bfa7e96dc0c99a70900413e469d6304344f8